inc/saiswitch.h

```diff
@@ -979,6 +993,119 @@ typedef enum _sai_switch_attr_t
     SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE,
 
     /**
+     * @brief Handle for switch profile id.
+     *
+     * Use this to retrive the Key-Vlaue pairs as part of switch 
+     * initialization. 
+     * @type sai_switch_profile_id_t
+     * @flags CREATE_ONLY
+     * @allownull true
+     */
+    SAI_SWITCH_ATTR_SWITCH_PROFILE_ID,
+
+    /**
+     * @brief Device Information for switch initialization.
+     * 
+     * Hardware information format is based on SAI implementations by vendors.
+     *   Example: Like PCI location, I2C adddress etc.
+     *
+     * @type sai_string_t
+     * @flags CREATE_ONLY
+     * @allownull true
+     */
+    SAI_SWITCH_ATTR_SWITCH_HARDWARE_INFO,
+
+    /**
+     * @brief Vendor specific path name of the firmware to load.
+     * 
+     * @type sai_string_t
+     * @flags CREATE_ONLY
+     * @allownull true
+     */
+    SAI_SWITCH_ATTR_FIRMWARE_PATH_NAME,
+
+    /**
+     * @brief Set to switch initialization or connect to NPU/SDK.
+     *
+     * TRUE - Initialize switch/SDK. 
+     * FALSE - Connect to SDK. This will connects library to the initialized SDK.
+     * After this call the capability attributes should be ready for retrieval
+     * via sai_get_switch_attribute()
+     *
+     * @type bool
+     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @default true
+     */
+    SAI_SWITCH_ATTR_INIT_SWITCH,
+
```

Comment:
Does it mean that to perform switch connect it is needed to pass existing switch id as pointer into create function ?
I really thing that we need to keep switch_connect/disconnect which will take switch id as value parameter.